packages/file-search/src/browser/quick-editors-open.ts

```diff
@@ -0,0 +1,191 @@
+};
+
+@injectable()
+export class QuickEditorsOpenService implements QuickOpenModel, QuickOpenHandler {
+
+    @inject(ApplicationShell)
+    protected readonly applicationShell: ApplicationShell;
+
+    @inject(KeybindingRegistry)
+    protected readonly keybindingRegistry: KeybindingRegistry;
+
+    @inject(FileSystem)
+    protected readonly fileSystem: FileSystem;
+
+    @inject(WorkspaceService)
+    protected readonly workspaceService: WorkspaceService;
+
+    @inject(NavigationLocationService)
+    protected readonly navigationLocationService: NavigationLocationService;
+
+    @inject(OpenerService)
+    protected readonly openerService: OpenerService;
+
+    @inject(PrefixQuickOpenService)
+    protected readonly quickOpenService: PrefixQuickOpenService;
+
+    @inject(FileSearchService)
+    protected readonly fileSearchService: FileSearchService;
+
+    @inject(MessageService)
+    protected readonly messageService: MessageService;
+
+    @inject(LabelProvider)
+    protected readonly labelProvider: LabelProvider;
+
+    protected isOpen: boolean = false;
+
+    readonly prefix: string = 'edt';
+
+    get description(): string {
+        return 'Show all editors';
+    }
+
+    getModel(): QuickOpenModel {
+        return this;
+    }
+
+    getOptions(): QuickOpenOptions {
+        return {
+            placeholder: '',
+            fuzzyMatchLabel: {
+                enableSeparateSubstringMatching: true
+            },
+            fuzzyMatchDescription: {
+                enableSeparateSubstringMatching: true
+            },
+            showItemsWithoutHighlight: true,
+            onClose: () => {
+                this.cancelIndicator.cancel();
+            }
+        };
+    }
+
+    isEnabled(): boolean {
+        return this.workspaceService.opened;
+    }
+
+    open(): void {
+        this.quickOpenService.open(this.prefix);
+    }
+
+    protected getKeyCommand(): string | undefined {
+        const keyCommand = this.keybindingRegistry.getKeybindingsForCommand(quickEditorsOpen.id);
+        if (keyCommand) {
+            // We only consider the first keybinding.
+            const accel = this.keybindingRegistry.acceleratorFor(keyCommand[0], '+');
+            return accel.join(' ');
+        }
+
+        return undefined;
+    }
+
+    private cancelIndicator = new CancellationTokenSource();
+
+    public async onType(lookFor: string, acceptor: (items: QuickOpenItem[]) => void): Promise<void> {
+        this.cancelIndicator.cancel();
+        this.cancelIndicator = new CancellationTokenSource();
+        const token = this.cancelIndicator.token;
+
+        const editorItems: QuickOpenItem[] = [];
+
+        // Get URIs of all EditorWidgets that are currently opened
+        // Sorted by URI
+        const widgets = this.applicationShell.widgets
+            .filter((widget): widget is EditorWidget => (widget instanceof EditorWidget))
+            .map(w => w.getResourceUri()!)
+            .sort();
+
```

Comment:
Add a check to determine if there are any `EditorWidgets` present, and display the message 
`List of opened editors is currently empty` if there aren't and return early.

Reply:
Sure ;)